de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554230_sys_gh6328_add_notification_logic_for_AttachmentListeners.sql

```diff
@@ -179,3 +179,8 @@ UPDATE AD_Message SET MsgText='Attached file was successfully imported.',Updated
 -- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
 UPDATE AD_Message_Trl SET MsgText='Attached file was successfully imported.',Updated=TO_TIMESTAMP('2020-03-10 17:13:20','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='en_US' AND AD_Message_ID=544969
 ;
+
+-- 2020-03-12T11:39:34.185Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Column SET MandatoryLogic='@IsSendNotification@=''Y''',Updated=TO_TIMESTAMP('2020-03-12 13:39:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=570102
```

Comment:
```suggestion
UPDATE AD_Column SET MandatoryLogic='@IsSendNotification/''N''@=''Y''',Updated=TO_TIMESTAMP('2020-03-12 13:39:34','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=570102
```
as common practice, add a default value such that if IsSendNotification is not in the ctx *for whatever reason* (e.g. no such AD_Field) there is no error. i don'T expect it to be needed in this particular case, but..yeah, as i said: common practice